config.py.example

```diff
@@ -101,3 +102,20 @@ else:
 # your own access token. See
 # help.github.com/articles/creating-an-access-token-for-command-line-use
 OAUTH_TOKEN = os.environ.get('OAUTH_TOKEN') or ""
+
+# Status categories used in the project
+# 'new', 'needsdiagnosis', 'needscontact', 'contactready' , 'sitewait', 'close'
+# Creating the model
+Category = namedtuple('Category', ['name', 'dataAttribute', 'label'])
+CATEGORIES = []
+cat_labels = [('new', 'new', 'New Issues'),
+              ('needsDiagnosis', 'needsdiagnosis', 'Needs Diagnosis'),
+              ('needsContact', 'needscontact', 'Needs Contact'),
+              ('ready', 'contactready', 'Ready for Outreach'),
+              ('sitewait', 'sitewait', 'Site Contacted'),
+              ('close', 'closed', 'Closed')]
```

Comment:
So it's basically "UI categories". I suggest clarifying the comment slightly, it's not about "status categories used in the project", it's only about enumerating the labels we want to flag in the UI as particular bug "states".